pilot/pkg/features/pilot.go

```diff
@@ -113,6 +113,18 @@ var (
 			" to unhealthy/non-ready hosts even if the percentage of healthy hosts fall below minimum health percentage(panic threshold).",
 	).Get())
 
+	PersistentSessionLabel = atomic.NewString(env.Register(
+		"PILOT_PERSISTENT_SESSION_LABEL",
+		"persistent-session",
```

Comment:
As discussed offline:

* label needs some dns name. Probably istio.io is easiest
* We need to verify we don't push on DRAINING (so it is zero cost). I think that is the case but we should verify. I will pont to the existing test we can expand tomorrow morning

Reply:
Fixed the names, and added an extra safety - if persistent session label is disabled we no longer track draining ( so no change in behavior if we also don't track unhealthy ).